src/applications/financial-status-report/components/EmploymentRecord.jsx

```diff
@@ -3,8 +3,10 @@ import { connect } from 'react-redux';
 import classNames from 'classnames';
 import PropTypes from 'prop-types';
 import { setData } from 'platform/forms-system/src/js/actions';
-import Select from '@department-of-veterans-affairs/component-library/Select';
-import MonthYear from '@department-of-veterans-affairs/component-library/MonthYear';
+import {
+  MonthYear,
+  Select,
+} from '@department-of-veterans-affairs/component-library';
```

Comment:
I changed the `MonthYear` import because it's using the same dynamically translated months as the `Date` component.

`MonthYear` uses `Select` internally, and `Select` is [generating a "unique" id](https://github.com/department-of-veterans-affairs/component-library/blob/29e1c0ce3a4771865630ff4c3243500d08019446/packages/react-components/src/components/Select/Select.jsx#L32) for itself. Having different `Select`s coming from different import locations means that the integers are no longer unique, so I had to change the `Select` import to keep things accessible.